toolboxes/fluid/cases/rigid_body_paper_tests/falling_ellipse/falling_ellipse.cfg

```diff
@@ -1,37 +1,39 @@
-directory=toolboxes/fluid/moving_body/three_sphere_2D/
+
+directory=toolboxes/fluid/new_testcases/falling_ellipse
 
 case.dimension=2
 
+
 [fluid]
 exporter.use-static-mesh=0
-filename=$cfgdir/three_sphere_2D.json
-mesh.filename=$cfgdir/three_sphere_2D.geo
-gmsh.hsize=3
-#solver=Oseen #Oseen,Picard,Newton
+filename=$cfgdir/falling_ellipse.json
+mesh.filename=$cfgdir/falling_ellipse.geo
+gmsh.hsize=0.009
+solver=Newton
 ksp-monitor=true
 pc-type=lu
 ksp-type=preonly
 #reuse-prec=1
 ksp-maxit-reuse=20
 snes-monitor=true
 snes-maxit=100
-define-pressure-cst=true
+#define-pressure-cst=true
 #define-pressure-cst.method=lagrange-multiplier#algebraic
 verbose_solvertimer=1
 
 #body.articulation.method=p-matrix
 
 [fluid.alemesh]
-
 pc-type=lu
 [fluid.bdf]
 order=2
 
 [ts]
-time-step=0.1#23
-time-final=20.1
-#restart=true
+order=2
+time-step=0.001# the time step should be verified
+time-final=2
 restart.at-last-save=true
-time-initial=-0.1#0#-0.1
-#save.freq=2
+time-initial=-0.001
```

Comment:
time initial is negatif? should be probably 0 (and not depends on the time step)

Reply:
fixed